ebsco/edsapi-koha-plugin
# Updates to handle custom links if not logged in

Allows custom links to work if you are not logged in.

To Test:
http://koha-2005.ebsco-gss.net/
Search "shakespeare"
Click on a PDF Link, note the prompt to login.
Click on a Linked Full Text link, note the prompt to login.
Filter to DOAJ (Directory of Open Access Journals)
Click on a DOAJ link
Note that you are not prompted to login
